client/src/main/resources/templates/js/main.js

```diff
@@ -394,9 +394,9 @@ function displaySource(src, title) {
         renderTable(pageAdditionalId, getFilterSearch(pageAdditionalId), getIndex(pageAdditionalId),
           getPageSize(pageAdditionalId));
 
-        var arrowDirection = '';
+        var arrowDirection = "";
         if(asc === "1") {
-          arrowDirection = 'dropup';
+          arrowDirection = "dropup";
         }
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1307322591/issues/source?bid=2928425&fileBranchId=2951451#l401)